src/components/Asset/AssetActions/Pool/Remove/index.module.css

```diff
@@ -50,25 +50,30 @@
 }
 
 .output {
-  composes: output from '../Add/Output.module.css';
+  border-top: 1px solid var(--border-color);
+  padding: var(--spacer);
+  padding-bottom: calc(var(--spacer) / 1.5);
+  display: grid;
+  gap: var(--spacer);
+  grid-template-columns: 1fr 1fr;
+  margin-left: -2rem;
+  margin-right: -2rem;
 }
 
-.output [class*='token'] {
-  white-space: nowrap;
+.output p {
+  font-weight: var(--font-weight-bold);
+  margin-bottom: calc(var(--spacer) / 8);
+  font-size: var(--font-size-small);
 }
 
-.output [class*='token'] > figure {
-  display: inline-block;
+.output [class*='token'] {
+  white-space: normal;
 }
 
-.output figure[class*='pool shares'] {
+.output [class*='token'] > figure {
```

Comment:
do we actually need this? As further down we put `noIcon` onto `Token` component (in code-review-only phase, did not check out visually yet)